project/plugins.sbt

```diff
@@ -1,6 +1,5 @@
 addSbtPlugin("com.github.sbt" % "sbt-release" % "1.1.0")
 
-addSbtPlugin("com.github.sbt" % "sbt-pgp" % "2.2.1")
+addSbtPlugin("ch.epfl.scala" % "sbt-version-policy" % "3.2.0")
```

Comment:
We don't need the `sbt-pgp` plugin, as [`gha-scala-library-release-workflow`](https://github.com/guardian/gha-scala-library-release-workflow) will sign the artifacts directly using GPG... but we _do_ need `sbt-version-policy`, to intelligently work out the correct version number based on compatibility with the latest release:

```sbt
releaseVersion := fromAggregatedAssessedCompatibilityWithLatestRelease().value
```